lib/stdlib/src/erl_lint.erl

```diff
@@ -4604,6 +4604,11 @@ check_load_nif(Anno, erlang, load_nif, [_, _], St0) ->
         true -> check_nif_inline(Anno, St);
         false -> St
     end;
+check_load_nif(Anno, erlang, nif_error, _, St) ->
+    case is_warn_enabled(nif_inline, St) of
+        true -> check_nif_inline(Anno, St);
+        false -> St
+    end;
```

Comment:
Nitpick. Since the logic is exactly the same as the case above, maybe it makes sense to refactor and avoid duplication.

```erlang
check_load_nif(Anno, erlang, load_nif, [_, _], St0) ->
    St = St0#lint{load_nif = true},
    warn_nif_inline(Anno, St);
check_load_nif(Anno, erlang, nif_error, _, St) ->
    warn_nif_inline(Anno, St).

warn_nif_inline(Anno, St) ->
  case is_warn_enabled(nif_inline, St) of
      true -> check_nif_inline(Anno, St);
      false -> St
  end.
```